compat/maven-model/src/main/java/org/apache/maven/model/io/xpp3/MavenXpp3Writer.java

```diff
@@ -43,12 +44,10 @@ public class MavenXpp3Writer {
     // - Methods -/
     // -----------/
 
-    public MavenXpp3Writer() {
-        this(false);
-    }
+    public MavenXpp3Writer() {}
 
-    protected MavenXpp3Writer(boolean addLocationInformation) {
```

Comment:
No API breakage in Maven 3 compatibility layer.

Reply:
Check. Won’t be able to work on this due to holiday for the next couple weeks. Feel free to adjust it, otherwise I’ll look into this when I’m back.